% Glossary entries

\newacronym{gls:ATP}{ATP}{Adenosine triphosphate}

\newacronym{gls:BLAST}{BLAST}{Basic Local Alignment Search Tool}

\newacronym{gls:CPR}{CPR}{Candidate Phyla Radiation}

\newacronym{gls:CSS}{CSS}{Cellular Signature Structure}

\newacronym{gls:DNA}{DNA}{Deoxyribonucleic Acid}

\newacronym{gls:dNTP}{dNTP}{Deoxynucleoside triphosphate}

\newacronym{gls:ESP}{ESP}{Eukaryotic Signature Protein}


\newacronym{gls:GI}{GI}{GenInfo Number}

\newacronym{gls:LAWKI}{LAWKI}{Life as we know it}

\newacronym{gls:LBA}{LBA}{Long Branch Attraction}

\newacronym{gls:LGT}{LGT}{Lateral Gene Transfer}

\newacronym{gls:LECA}{LECA}{Last Eukaryotic Common Ancestor}

\newacronym{gls:LUCA}{LUCA}{Last Universal Common Ancestor}

\newacronym{gls:ML}{ML}{Maximum Likelihood}

\newacronym{gls:MMR}{MMR}{DNA Mismatch Repair}

\newacronym{gls:MP}{MP}{Maximum Parsimony}

\newacronym[see={[Glossary:]{gls:nucleoside-triphosphate}}]{gls:NTP}{NTP}{Nucleoside triphosphate\glsadd{gls:nucleoside-triphosphate}}

\newacronym{gls:RNA}{RNA}{Ribonucleic Acid}

\newacronym{gls:SFI}{SFI}{Santa Fe Institute}

\newacronym{gls:TOL}{TOL}{Tree of Life}

\newglossaryentry{gls:16S-ribosomal-RNA}{
	name={16S ribosomal RNA},
	description={16S ribosomal RNA (or 16S rRNA) is the component of the 30S small subunit of a prokaryotic ribosome that binds to the Shine-Dalgarno sequence. The genes coding for it are referred to as 16S rRNA gene and are used in reconstructing phylogenies, due to the slow rates of evolution of this region of the gene. Carl Woese and George E. Fox were two of the people who pioneered the use of 16S rRNA in phylogenetics in 1977--\cite{wiki:16SrRNA}.}}

\newglossaryentry{gls:aldehyde}{
	name={aldehyde},
	description={An aldehyde  is a compound containing a functional group with the structure -CHO, consisting of a carbonyl center (a carbon double-bonded to oxygen) with the carbon atom also bonded to hydrogen and to an R group, which is any generic alkyl or side chain--\cite{wiki:aldehyde} }}

\newglossaryentry{gls:aldose} {
	name={aldose},
	description={An aldose is a monosaccharide (a simple sugar) with a carbon backbone chain with a carbonyl group on the endmost carbon atom, making it an aldehyde, and hydroxyl groups connected to all the other carbon atoms. Aldoses can be distinguished from \glspl{gls:ketose}, which have the carbonyl group away from the end of the molecule, and are therefore ketones--\cite{ wiki:aldose}}
}

\newglossaryentry{gls:amphiphile}{
	name={amphiphile},
	description={An amphiphile is a chemical compound possessing both hydrophilic (water-loving, polar) and lipophilic (fat-loving) properties. Such a compound is called amphiphilic or amphipathic--\cite{wiki:amphiphile}}}

\newglossaryentry{gls:arrheniusequation}{
	name={Arrhenius Equation},
	description={In physical chemistry, the Arrhenius equation is a formula for the temperature dependence of reaction rates. The equation was proposed by Svante Arrhenius in 1889, based on the work of Dutch chemist Jacobus Henricus van 't Hoff who had noted in 1884 that van 't Hoff equation for the temperature dependence of equilibrium constants suggests such a formula for the rates of both forward and reverse reactions--\cite{wiki:arrheniusequation}
	\begin{align*}
	k =& e^{\frac{-E_a}{k_B T}}\text{, where}\\
	k =& \text{rate constant,}\\
	T =& \text{the absolute temperature (in Kelvin),}\\
	A =& \text{the pre-exponential factor, a constant for each chemical reaction. According to collision theory, A is the frequency of collisions in the correct orientation,}\\
	E_a =& \text{the activation energy for the reaction,}\\
	k_B =& \text{the Boltzmann constant.}
	\end{align*}
}}

\newglossaryentry{gls:autocatalysis}{
	name={autocatalysis},
	description={Chemical self production: the ability of a set of chemical species to make more of that same set, by undergoing a series of chemical 	reactions--\cite{sfi2019}}}

\newglossaryentry{gls:autocatalyst}{
	name={autocatalyst},
	description={Something that acts as a \gls{gls:catalyst} for its own production}}

\newglossaryentry{gls:autotroph}{
	name={autotroph},
	description={An autotroph or primary producer, is an organism that produces complex organic compounds (such as carbohydrates, fats, and proteins) from simple substances present in its surroundings, generally using energy from light (photosynthesis) or inorganic chemical reactions (chemosynthesis). They are the producers in a food chain, such as plants on land or algae in water (in contrast to \glspl{gls:heterotroph} as consumers of autotrophs). They do not need a living source of energy or organic carbon. Autotrophs can reduce carbon dioxide to make organic compounds for biosynthesis and also create a store of chemical energy. Most autotrophs use water as the reducing agent, but some can use other hydrogen compounds such as hydrogen sulfide. Some autotrophs, such as green plants and algae, are phototrophs, meaning that they convert electromagnetic energy from sunlight into chemical energy in the form of reduced carbon--\cite{wiki:autotroph} }}

\newglossaryentry{gls:Candidate:Phyla:Radiation}{
	name={Candidate Phyla Radiation},
	description={The Candidate Phyla Radiation (CPR) comprises a huge group of bacteria that have small genomes that rarely encode CRISPR-Cas systems for phage defense--\cite{chen2019candidate}}}

\newglossaryentry{gls:candidatus}{
	name={candidatus},
	description={In prokaryote nomenclature, Candidatus (Latin for candidate of Roman office, named after the white gown worn by Roman senators) is a component of the taxonomic name for a bacterium or other prokaryote, that cannot be maintained in a microbiological culture collection. It is an interim taxonomic status for yet-to-be-cultured microorganisms--\cite{wiki:2018candidatus}''..., we propose the category Candidatus (L. candidatus, a candidate, to indicate that the assignment is provisional) followed by a descriptive vernac- ular epithet and then the available  essential  information''--\cite{murray1994taxonomic}}}

\newglossaryentry{gls:catalyst}{
	name={catalyst},
	description={Something that gets returned after a sequence 	of reactions}}

\newglossaryentry{gls:cellular:signature:structures}{
	name={Cellular Signature Structures},
	description={A cellular compartment that  distinguished  eukaryotes  from  prokaryotes.\cite{kurland2006genomics}}}

\newglossaryentry{gls:clathrate}{
	name={clathrate},
	description={Inclusion compound in which the guest molecule is in a cage
		formed by the host molecule or by a lattice of host molecules--\cite{iupac2009goldbook}}}

\newglossaryentry{gls:coacervate}{
	name={coacervate},
	description={The separation into two liquid phases in colloidal
		systems. The phase more concentrated in colloid
		component is the coacervate, and the other phase is the equilibrium solution--\cite{iupac2009goldbook}}}
	
\newglossaryentry{gls:CpG}{
	name={CpG},
	description={The CpG sites or CG sites are regions of DNA where a cytosine nucleotide is followed by a guanine nucleotide in the linear sequence of bases along its 5' â†’ 3' direction. CpG sites occur with high frequency in genomic regions called CpG islands (or CG islands). Cytosines in CpG dinucleotides can be methylated to form 5-methylcytosines. Enzymes that add a methyl group are called DNA methyltransferases. In mammals, 70\% to 80\% of CpG cytosines are methylated. Methylating the cytosine within a gene can change its expression, a mechanism that is part of a larger field of science studying gene regulation that is called epigenetics\cite{wiki:CpG}}}

\newglossaryentry{gls:deamination}{
	name={deamination},
	description={Deamination is the removal of an amino group from a molecule. Enzymes that catalyse this reaction are called deaminases--\cite{wiki:deamination}}}

\newglossaryentry{gls:DNAmethyltransferase}
	{name={DNAmethyltransferase},
	description={In biochemistry, the DNA methyltransferase (DNA MTase) family of enzymes catalyze the transfer of a methyl group to DNA. DNA methylation serves a wide variety of biological functions. All the known DNA methyltransferases use S-adenosyl methionine (SAM) as the methyl donor--\cite{wiki:DNAmethyltransferase}}}

\newglossaryentry{gls:endomembrane}{
	name={endomembrane},
	description={The endomembrane system is composed of the different membranes that are suspended in the cytoplasm within a eukaryotic cell. These membranes divide the cell into functional and structural compartments, or organelles. In eukaryotes the organelles of the endomembrane system include: the nuclear membrane, the endoplasmic reticulum, the Golgi apparatus, lysosomes, vesicles, endosomes, and plasma (cell) membrane among others. The system is defined more accurately as the set of membranes that form a single functional and developmental unit, either being connected directly, or exchanging material through vesicle transport. Importantly, the endomembrane system does not include the membranes of chloroplasts or mitochondria, but might have evolved from the latter--\cite{wiki:endomembrane}}}

\newglossaryentry{gls:enthalpy}{
	name={enthalpy},
	description={Internal energy
		of a system plus the product of pressure and volume. Its change in a system is equal to the heat brought to the system at constant pressure.--\cite{iupac2009goldbook}}}

\newglossaryentry{gls:eocyte}{
	name={eocyte},
	description={Eocytes are sulphur-metabolizing, extreme thermophile organisms, placed within the Crenarchaeota in the three-domain taxonomic system, but recent genetic studies have shown eocytes to be very closely related to eukaryotes, suggesting that eukaryotes and Crenarchaeota share a common ancestor. This evidence supports the abandonment of the three-domain classification in favour of a two-domain classification comprising the Eubacteria and Archaea, with the Eukarya contained within the Archaea--\cite{allaby2010eocyte}}}

\newglossaryentry{gls:gammaproteobacteria}{
	name={gammaproteobacteria},
	description={Gammaproteobacteria are a class of bacteria. Several medically, ecologically, and scientifically important groups of bacteria belong to this class. Like all Proteobacteria, the Gammaproteobacteria are Gram-negative. --\cite{wiki:gammaproteobacteria}}}


\newglossaryentry{gls:eocyte:hypothesis}{
	name={eocyte},
	description={A proposed taxonomic revision that would replace the three-*domain classification with one of only two domains, Bacteria and Archaea, an idea first proposed in 1984 by James Lake and colleagues. Eocytes are sulphur-metabolizing, extreme thermophile organisms, placed within the Crenarchaeota in the three-domain taxonomic system, but recent genetic studies have shown eocytes to be very closely related to eukaryotes, suggesting that eukaryotes and Crenarchaeota share a common ancestor. This evidence supports the abandonment of the three-domain classification in favour of a two-domain classification comprising the Eubacteria and Archaea, with the Eukarya contained within the Archaea--\cite{allaby2010eocyte}}}

\newglossaryentry{gls:eukaryotic:signature:protein}{
	name={Eukaryotic Signature Protein},
	description={A protein that is found in eukaryotic cells but has no significant homology to proteins in Archaea and Bacteria--cite{hartman2002origin}}}

\newglossaryentry{gls:furan}{
	name={furan},
	description={Furan is a heterocyclic organic compound, consisting of a five-membered aromatic ring with four carbon atoms and one oxygen. Chemical compounds containing such rings are also referred to as furans--\cite{wiki:furan}}
}

\newglossaryentry{gls:gibbs-free}{
	name={Gibbs free energy},
	description={A thermodynamic potential that can be used to calculate the maximum of reversible work that may be performed by a thermodynamic system at a constant temperature and pressure (isothermal, isobaric). The Gibbs free energy is the maximum amount of non-expansion work that can be extracted from a thermodynamically closed system (one that can exchange heat and work with its surroundings, but not matter); this maximum can be attained only in a completely reversible process--\cite{wiki:gibbs-free}}}



\newglossaryentry{gls:heterotachy}{
	name={heterotachy},
	description={Heterotachy refers to variations in lineage-specific evolutionary rates over time. In the field of molecular evolution, the principle of heterotachy states that the substitution rate of sites in a gene can change through time--\cite{lopez2002heterotachy}}}

\newglossaryentry{gls:heterotroph}{
	name={heterotroph},
	description={A heterotroph is an organism that cannot produce its own food, relying instead on the intake of nutrition from other sources of organic carbon, mainly plant or animal matter. In the food chain, heterotrophs are primary, secondary and tertiary consumers, but not producers. Living organisms that are heterotrophic include all animals and fungi, some bacteria and protists, and parasitic plants. The term heterotroph arose in microbiology in 1946 as part of a classification of microorganisms based on their type of nutrition. The term is now used in many fields, such as ecology in describing the food chain--\cite{wiki:heterotroph} }}

\newglossaryentry{gls:hydrogenosome }{
	name={hydrogenosome },
	description={A hydrogenosome is a membrane-enclosed organelle of some anaerobic ciliates, trichomonads, fungi, and animals. The hydrogenosomes of trichomonads (the most studied of the hydrogenosome-containing microorganisms) produce molecular hydrogen, acetate, carbon dioxide and ATP by the combined actions of pyruvate:ferredoxin oxido-reductase, hydrogenase, acetate:succinate CoA transferase and succinate thiokinase. Superoxide dismutase, malate dehydrogenase (decarboxylating), ferredoxin, adenylate kinase and NADH:ferredoxin oxido-reductase are also localized in the hydrogenosome. It is nearly universally accepted that hydrogenosomes evolved from mitochondria--\cite{wiki:hydrogenosome}}}

\newglossaryentry{gls:isolated:representatives}{
	name={isolated representatives},
	description={(of a phylum) one or more species have been cultured individually in the laboratory--\cite{taylor2016branching}.}}

\newglossaryentry{gls:ketone}{
	name={ketone},
	description={In chemistry, a ketone is a functional group with the structure RC(=O)R', where R and R' can be a variety of carbon-containing substituents. Ketones contain a carbonyl group (a carbon-oxygen double bond)--\cite{wiki:ketone} }}

\newglossaryentry{gls:ketose}{
	name={ketose},
	description={A ketose is a monosaccharide containing one ketone group per molecule. The simplest ketose is dihydroxyacetone, which has only three carbon atoms, and it is the only one with no optical activity. All monosaccharide ketoses are reducing sugars, because they can tautomerize into \glspl{gls:aldose} via an aldol intermediate, and the resulting aldehyde group can be oxidised, for example in the Tollens' test or Benedict's test. Ketoses that are bound into glycosides, for example in the case of the fructose moiety of sucrose, are nonreducing sugars--\cite{wiki:ketose}}
}

\newglossaryentry{gls:kinetic:isotope:effect}{
	name={kinetic isotope effect},
	description={In physical organic chemistry, a kinetic isotope effect (KIE) is the change in the reaction rate of a chemical reaction when one of the atoms in the reactants is replaced by one of its isotopes. Formally, it is the ratio of rate constants for the reactions involving the light $k_L$ and the heavy $k_H$ (isotopically substituted reactants (isotopologues) $\frac{k_L}{k_H}$\cite{wiki:kinetic:isotope:effect}}}


\newglossaryentry{gls:kinetics}{
	name={kinetics},
	description={Concerning the rate of transitioning between states.
    Kinetic stability refers to being in a state which transitions slowly.}}

\newglossaryentry{gls:landauer_bound}{
	name={Landauer Bound},
	description={Landauer Bound: any logically irreversible manipulation of information, such as the erasure of a bit or the merging of two computation paths, must be accompanied by a corresponding entropy increase in non-information-bearing degrees of freedom of the information-processing apparatus or its environment--$kT(H_i - H_f)$, where $H_i$ represents the entropy of the initial state, $H_i$ of the final\cite{wiki:landauer}}}

\newglossaryentry{gls:life}{
	name={Life},
	description={Life is a self-propagating chemical system capable of undergoing adaptive evolution--adapted from \cite{deamer1994origins}}}

\newglossaryentry{gls:lipid}{
	name={lipid},
	description={Lipids are amphiphilic molecules that  self-assemble into a hydrophobic barrier that surrounds a cell\cite[Lecture 2.7.1]{sfi2019}}}

\newglossaryentry{gls:lokiarchaeota}{
	name={Lokiarchaeota},
	description={Lokiarchaeota is a proposed phylum of the Archaea. The phylum includes all members of the group previously named Deep Sea Archaeal Group (DSAG), also known as Marine Benthic Group B (MBG-B). A phylogenetic analysis disclosed a monophyletic grouping of the Lokiarchaeota with the eukaryotes. The analysis revealed several genes with cell membrane-related functions. The presence of such genes support the hypothesis of an archaeal host for the emergence of the eukaryotes; the eocyte-like scenarios --cite{wiki:lokiarchaeota} }}

\newglossaryentry{gls:long:branch:attraction}{
	name={long branch attraction},
	description={In phylogenetics, long branch attraction is a form of systematic error whereby distantly related lineages are incorrectly inferred to be closely related. Long branch attraction arises when the amount of molecular or morphological change accumulated within a lineage is sufficient to cause that lineage to appear similar (thus closely related) to another long-branched lineage, solely because they have both undergone a large amount of change, rather than because they are related by descent. Such bias is more common when the overall divergence of some taxa results in long branches within a phylogeny. Long-branches are often attracted to the base of a phylogenetic tree, because the lineage included to represent an outgroup is often also long-branched--\cite{wiki:long:branch:attraction}}}

\newglossaryentry{gls:methylation}{
	name={methylation},
	description={DNA methylation is a biological process by which methyl groups are added to the DNA molecule. Methylation can change the activity of a DNA segment without changing the sequence. When located in a gene promoter, DNA methylation typically acts to repress gene transcription. In mammals, DNA methylation is essential for normal development and is associated with a number of key processes including genomic imprinting, X-chromosome inactivation, repression of transposable elements, aging, and carcinogenesis--\cite{wiki:methylation} }}

\newglossaryentry{gls:mitosome}{
	name={mitosome},
	description={An organelle found in "amitochondrial" unicellular organisms which do not have the capability of gaining energy from oxidative phosphorylation. Mitosomes are almost certainly derived from mitochondria, they have a double membrane and most proteins are delivered to them by a targeting sequence. Unlike mitochondria, mitosomes do not contain any DNA. The mitosome functions in iron-sulphur cluster assembly--\cite{uniprot2009mitosome}}}

\newglossaryentry{gls:nucleolus}{
	name={nucleolus},
	plural={nucleoli},
	description={The nuclear site of rRNA transcription, processing, and ribosome assembly\cite{cooper2000sinauer}}}

\newglossaryentry{gls:nucleoside-triphosphate}{
	name={nucleoside triphosphate},
	description={A nucleoside triphosphate is a molecule containing a nitrogenous base bound to a 5-carbon sugar (either ribose or deoxyribose), with three phosphate groups bound to the sugar\cite{wiki:nucleoside:triphosphate}}}

\newglossaryentry{gls:oligotroph}{
	name={oligotroph},
	description={An oligotroph is an organism that can live in an environment that offers very low levels of nutrients. They may be contrasted with copiotrophs, which prefer nutritionally rich environments. Oligotrophs are characterized by slow growth, low rates of metabolism, and generally low population density --\cite{wiki:oligotroph}}}

\newglossaryentry{gls:orthology}{
	name={orthology},
	description={Where the homology is the result of speciation so that the history of the gene reflects the history of the species (for example $\alpha$ haemoglobin in man and mouse) the genes should be called \textit{orthologous}--\cite{fitch1970distinguishing}}}

\newglossaryentry{gls:outgroup}{
	name={outgroup},
	description={In cladistics or phylogenetics, an outgroup is a more distantly related group of organisms that serves as a reference group when determining the evolutionary relationships of the ingroup, the set of organisms under study, and is distinct from sociological outgroups. The outgroup is used as a point of comparison for the ingroup and specifically allows for the phylogeny to be rooted. Because the polarity (direction) of character change can be determined only on a rooted phylogeny, the choice of outgroup is essential for understanding the evolution of traits along a phylogeny\cite{wiki:outgroup}}}

\newglossaryentry{gls:parology}{
	name={parology},
	description={Where the hology is the result of gene duplication so that both copies have descended side by side during the history of an organism (for example $\alpha$ and $\beta$ hemoglobin)  the genese should be called paralogous\cite{fitch1970distinguishing}}}

\newglossaryentry{gls:phospholipid}{
	name={phospholipid},
	description={Phospholipids (PL) are a class of lipids that are a major component of all cell membranes. They can form lipid bilayers because of their amphiphilic characteristic. The structure of the phospholipid molecule generally consists of two hydrophobic fatty acid "tails" and a hydrophilic "head" consisting of a phosphate group. The two components are usually joined together by a glycerol molecule. The phosphate groups can be modified with simple organic molecules such as choline, ethanolamine or serine\cite{wiki:phospholipid}}}

\newglossaryentry{gls:phylogenetics}{
	name={phylogenetics},
	description={In biology, phylogenetics  is the study of the evolutionary history and relationships among individuals or groups of organisms (e.g. species, or populations). These relationships are discovered through phylogenetic inference methods that evaluate observed heritable traits, such as DNA sequences or morphology under a model of evolution of these traits. The result of these analyses is a phylogeny (also known as a phylogenetic tree) â€“ a diagrammatic hypothesis about the history of the evolutionary relationships of a group of organisms. The tips of a phylogenetic tree can be living organisms or fossils, and represent the "end", or the present, in an evolutionary lineage. A phylogenetic tree can be rooted or unrooted. A rooted tree indicates the common ancestor, or ancestral lineage, of the tree. An unrooted tree makes no assumption about the ancestral line, and does not show the origin or root of the gene or organism in question. Phylogenetic analyses have become central to understanding biodiversity, evolution, ecology, and genomes\cite{wiki:phylogenetics}}}


\newglossaryentry{gls:poza}{
	name={poza},
	description={Earth bordered basins for water (commonly used for irrigation in the Andes)--\cite{denevan2003cultivated}}}



\newglossaryentry{gls:Proteobacteria}{
	name={proteobacteria},
	description={Proteobacteria is a major phylum of gram-negative bacteria. They include a wide variety of pathogens, such as Escherichia, Salmonella, Vibrio, Helicobacter, Yersinia, Legionellales and many other notable genera. Others are free-living (non-parasitic) and include many of the bacteria responsible for nitrogen fixation.--\cite{stackebrandt1988proteobacteria}}}

\newglossaryentry{gls:price:equation}{
	name={Price equation},
	description={In the theory of evolution and natural selection, the Price equation (also known as Price's equation or Price's theorem) describes how a trait or gene changes in frequency over time. The equation uses a covariance between a trait and fitness, to give a mathematical description of evolution and natural selection. It provides a way to understand the effects that gene transmission and natural selection have on the proportion of genes within each new generation of a population. The Price equation was derived by George R. Price, working in London to re-derive W.D. Hamilton's work on kin selection. Examples of the Price equation have been constructed for various evolutionary cases. The Price equation also has applications in economics--\cite{ wiki:price:equation}}}

\newglossaryentry{gls:purine}{
	name={purine},
	description={Purine is a heterocyclic aromatic organic compound that consists of a pyrimidine ring fused to an imidazole ring. It is water-soluble. Purine also gives its name to the wider class of molecules, purines, which include substituted purines and their tautomers. They are the most widely occurring nitrogen-containing heterocycles in nature--\cite{wiki:purine}}}



\newglossaryentry{gls:PVC}{
	name={PVC superphylum},
	description={The PVC superphylum is a grouping of distinct phyla of the domain bacteria proposed initially on the basis of 16S rRNA gene sequence analysis. It consists of a core of phyla Planctomycetes, Verrucomicrobia and Chlamydiae, but several other phyla have been considered to be members, including phylum Lentisphaerae and several other phyla consisting only of yet-to-be cultured members. The genomics-based links between Planctomycetes, Verrucomicrobia and Chlamydiae have been recently strengthened, but there appear to be other features which may confirm the relationship at least of Planctomycetes, Verrucomicrobia and Lentisphaerae. Remarkably these include the unique planctomycetal compartmentalized cell plan differing from the cell organization typical for bacteria. Such a shared cell plan suggests that the common ancestor of the PVC superphylum members may also have been compartmentalized, suggesting this is an evolutionarily homologous feature at least within the superphylum. Both the PVC endomembranes and the eukaryote-homologous membrane-coating MC proteins linked to endocytosis ability in Gemmata obscuriglobus and shared by PVC members suggest such homology may extend beyond the bacteria to the Eukarya. If so, either our definition of bacteria may have to change or PVC members admitted to be exceptions.--\cite{fuerst2013pvc}}}

\newglossaryentry{gls:pyran}{
	name={pyran},
	description={In chemistry, pyran, or oxine, is a six-membered heterocyclic, non-aromatic ring, consisting of five carbon atoms and one oxygen atom and containing two double bonds. The molecular formula is C5H6O. There are two isomers of pyran that differ by the location of the double bonds. In 2H-pyran, the saturated carbon is at position 2, whereas, in 4H-pyran, the saturated carbon is at position 4--\cite{wiki:pyran} }
}

\newglossaryentry{gls:pyrimidine}{
	name={pyrimidine},
	description={Pyrimidine is an aromatic heterocyclic organic compound similar to pyridine. One of the three diazines (six-membered heterocyclics with two nitrogen atoms in the ring), it has the nitrogen atoms at positions 1 and 3 in the ring. The other diazines are pyrazine (nitrogen atoms at the 1 and 4 positions) and pyridazine (nitrogen atoms at the 1 and 2 positions). In nucleic acids, three types of nucleobases are pyrimidine derivatives: cytosine (C), thymine (T), and uracil (U)--\cite{wiki:pyrimidine}}}

\newglossaryentry{gls:r-K-selection}{
	name={r/K selection},
	description={In ecology, r/K selection theory relates to the selection of combinations of traits in an organism that trade off between quantity and quality of offspring. The focus on either an increased quantity of offspring at the expense of individual parental investment of r-strategists, or on a reduced quantity of offspring with a corresponding increased parental investment of K-strategists, varies widely, seemingly to promote success in particular environments--\cite{wiki:k-r-selection}}}


\newglossaryentry{gls:reaction-network}{
	name={reaction network},
	description={multiple chemical reactions that interact}}

\newglossaryentry{gls:ribozyme}{
	name={ribozyme},
	description={Ribozymes (ribonucleic acid enzymes) are RNA molecules that are capable of catalyzing specific biochemical reactions, similar to the action of protein enzymes. The 1982 discovery of ribozymes demonstrated that RNA can be both genetic material (like DNA) and a biological catalyst (like protein enzymes), and contributed to the RNA world hypothesis, which suggests that RNA may have been important in the evolution of prebiotic self-replicating systems. The most common activities of natural or in vitro-evolved ribozymes are the cleavage or ligation of RNA and DNA and peptide bond formation. Within the ribosome, ribozymes function as part of the large subunit ribosomal RNA to link amino acids during protein synthesis. They also participate in a variety of RNA processing reactions, including RNA splicing, viral replication, and transfer RNA biosynthesis. Examples of ribozymes include the hammerhead ribozyme, the VS ribozyme, Leadzyme and the hairpin ribozyme--\cite{wiki:ribozyme} }}

\newglossaryentry{gls:saprotroph}{
	name={saprotroph},
	description={Saprotrophic nutrition  or lysotrophic nutrition is a process of chemoheterotrophic extracellular digestion involved in the processing of decayed (dead or waste) organic matter. It occurs in saprotrophs, and is most often associated with fungi (for example Mucor) and soil bacteria--\cite{wiki:saprotroph}}}

\newglossaryentry{gls:serpentinization}{
	name={serpentinization},
	description={A hydration and metamorphic transformation of ultramafic rock from the Earth's mantle--\cite{wiki:serpentinization}}
}

\newglossaryentry{gls:spliceosome}{
	name={spliceosome},
	description={Large complex of snRNAs and proteins that catalyzes the splicing of pre-mRNAs--\cite{cooper2000sinauer}}}

\newglossaryentry{gls:sympatric-speciation}{
	name={sympatric speciation},
	description={the evolution of a new species from a surviving ancestral species while both continue to inhabit the same geographic region--\cite{wiki:Sympatric:speciation}}}

\newglossaryentry{gls:symplesiomorphy}{
	name={symplesiomorphy},
	description={In phylogenetics, a plesiomorphy, symplesiomorphy or symplesiomorphic character is an ancestral character (trait state) shared by two or more taxa - but also with other taxa linked earlier in the clade (that is, having an earlier last common ancestor, with them, than theirs)--\cite{wiki:symplesiomorphy} }}

\newglossaryentry{gls:synapomorphy}{
	name={synapomorphy},
	description={In phylogenetics, apomorphy and synapomorphy refer to derived characters of a clade: characters or traits that are derived from ancestral characters over evolutionary history. An apomorphy is a character that is different from the form found in an ancestor, i.e., an innovation, that sets the clade apart from other clades. A synapomorphy is a shared apomorphy that distinguishes a clade from other organisms. In other words, it is an apomorphy shared by members of a monophyletic group, and thus assumed to be present in their most recent common ancestor--cite{ wiki:synapomorphy } }}

\newglossaryentry{gls:thermodynamics}{
	name={thermodynamics},
	description={Concerning the flow of energy between states. Thermodynamic stability refers to being in a low energy state}}

\newglossaryentry{gls:zwitterion}{
	name={zwitterion},
	description={In chemistry, a zwitterion is a molecule with two or more functional groups, of which at least one has a positive and one has a negative electrical charge and the net charge of the entire molecule is zero--\cite{wiki:zwitterion}}}