\documentclass[]{article}
\usepackage{caption,subcaption,graphicx,float,url,amsmath,amssymb,tocloft}
\usepackage[hidelinks]{hyperref}
\usepackage[toc,acronym,nonumberlist]{glossaries}
\usepackage{titling}
\setacronymstyle{long-short}
\usepackage{glossaries-extra}
\graphicspath{{figs/}} 
\setlength{\cftsubsecindent}{0em}
\setlength{\cftsecnumwidth}{3em}
\setlength{\cftsubsecnumwidth}{3em}
\pretitle{
	\begin{center}
		\includegraphics[width=6cm]{KanjiLife}\\
		
	}
	\posttitle{\end{center}}

%opening
\title{
	Notes from Origins of Life\\
	Week 5: Evolution}
\author{Simon Crase (compiler)\\simon@greenweaves.nz}

\makeglossaries

\loadglsentries{glossary-entries}

\renewcommand{\thesection}{5.\arabic{section}}
\renewcommand{\glstextformat}[1]{\textbf{\em #1}}
\newcommand{\E}{\mathrm{E}}
\newcommand{\Var}{\mathrm{Var}}
\newcommand{\Cov}{\mathrm{Cov}} 
\newcommand\numberthis{\addtocounter{equation}{1}\tag{\theequation}}

\begin{document}

\maketitle

\begin{abstract}
   These are my notes from the $5^{th}$ Week of the Santa Fe Institute Origins of Life Course\cite{sfi2020}. 
   The content and images contained herein are the intellectual property of the Santa Fe Institute, with the exception of any errors in transcription, which are my own.
   These notes are distributed in the hope that they will be useful,
   but without any warranty, and without even the implied warranty of
   merchantability or fitness for a particular purpose. All feedback is welcome,
   but I don't necessarily undertake to do anything with it.
\end{abstract}

\setcounter{tocdepth}{2}
\tableofcontents
\listoffigures

\section[Introduction]{Introduction--Chris Kempes}

We discuss Selection, \gls{gls:phylogenetics}, Macroscopic Regularities across all of life, how  life has evolved in creating Complexity, and evolutionary processes in Artificial Life. 

\section[Origins of Eukaryotes]{Origins of Eukaryotes--David Baum}

Since life originated, perhaps 4 billion years ago, on this planet, most people would say that the most important subsequent transition was the evolutionary origin of the eukaryotes--the origin of large, complex cells such as we find in the bodies of animals, plants, and fungi. The origin lf eukaryotes is a distinct problem to the origin of life, but it shares the features that we don't have intermediate steps, and it happened just once. It can serve as a model for studying difficult and rare transitions.

Cellular life began \gls{gls:prokaryotic}-- \glsdesc{gls:prokaryotic}
boundary--Figure \ref{fig:prokaryote}. Structurally \gls{gls:prokaryotic} cells are very simple, but we must remember that the \gls{gls:cytoplasm} is very complicated at the biochemical and genetic level.

\begin{figure}[H]
	\begin{center}
		\caption{Cellular life began \gls{gls:prokaryotic}}\label{fig:prokaryote}
		\includegraphics[width=0.6\textwidth]{prokaryote}
	\end{center}
\end{figure}

\Gls{gls:eukaryotic} cells are structurally complex. 

\begin{itemize}
	\item They are larger
	\item They have multiple internal membrane-bound structures
	\begin{itemize}
		\item \emph{nucleus} enclosed genetic material
		\item \emph{mitochondria} generate energy for the cell
		\item endomembrane system
	\end{itemize}
	\item A significant evolutionary event!
\end{itemize}

So \gls{gls:eukaryotic} cells are significantly different from \gls{gls:prokaryotic} cells, and we have no intermediates known between these two states of being.
\begin{itemize}
	\item Cellular predators--Figure \ref{figs:Cellular:predators:symbiotic:hosts1}--the lions and tigers of the micro-biotic world!
	\item Symbiotic hosts--Figure \ref{figs:Cellular:predators:symbiotic:hosts2}-- the farmers of the micro-biotic world!
	\item Multicellularity
	\begin{itemize}
		\item why have only eukaryotes evolved multicellularity?
		\item Why have eukaryotes evolved it multiple times? The best known instances are animals, plants, and fungi, but there are others.
	\end{itemize}
\end{itemize}

\begin{figure}[H]
	\caption{Cellular predators, symbiotic hosts}
	\label{figs:Cellular:predators:symbiotic:hosts}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{ }
		\label{figs:Cellular:predators:symbiotic:hosts1}
		\includegraphics[width=\textwidth]{Eukaryotes1}
	\end{subfigure}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{ }
		\label{figs:Cellular:predators:symbiotic:hosts2}
		\includegraphics[width=\textwidth]{Eukaryotes2}
	\end{subfigure}
\end{figure}
Why eukaryotes? There are three ideas at present.

\begin{itemize}
	\item Mitochondria improve energetic efficiency, and allow energy use to be regulated more nimbly.\cite{lane2010energetics}.
	
	\item Flexible membranes allow phagocytosis and generation of internal vesicles. So eukaryotes can do things that prokaryotes can't, such as engulfing food prey--Figure \ref{fig:engulf}.
	
	\item Nucleus and \gls{gls:endomembrane} system allow for finer gene
	regulation--Figure \ref{fig:finer:gene:regulation}\cite{paez2016endocytosis}. 
	\begin{itemize}
		\item Multicellular organization requires communications between different cells, which express different genes and proteins.
		\item The extra layers of regulation that eukaryotes have may explain why they, and not prokaryotos, have evolved complex multicellular organisms. These layers deal with the genetic control by separating the transcription of a DNA sequence from the reading of the message--there are many steps where regulation can be imposed. 
		\item Additionally the \gls{gls:endomembrane} system can affect the secretion, degradation and recycling of proteins
	\end{itemize}
\end{itemize}

\begin{figure}[H]
	\caption{Flexible membranes allow phagocytosis and generation
		of internal vesicles}
	\label{fig:engulf}
	\includegraphics[width=0.9\textwidth]{Engulf}
\end{figure}


\begin{figure}[H]
	\caption{Nucleus and endomembrane system allow for finer gene
		regulation}\label{fig:finer:gene:regulation}
	\begin{subfigure}[b]{0.45\textwidth}
		\includegraphics[width=\textwidth]{Regulation1}
	\end{subfigure}
	\begin{subfigure}[b]{0.45\textwidth}
	\includegraphics[width=\textwidth]{Regulation2}
	\end{subfigure}
\end{figure}
Easier to study than the
origin of life

\begin{itemize}
	\item We can use phylogenetic methods to infer the features of \gls{gls:LECA}--Figure \ref {fig:LECA}. We can study both ends of the \gls{gls:LECA} branch, since we have procaryotes; we cannot do this with \gls{gls:LUCA}.
	\begin{itemize}
		\item Mitochondria are endosymbiotic bacteria\cite{germot1996presence}
		\item Host was an archaeon\cite{spang2015complex}
		\item Can distinguish genes of archaeal or
		bacterial ancestry\cite{thiergart2012evolutionary}
	\end{itemize}
\end{itemize}

\begin{figure}[H]
	\caption{LECA}\label{fig:LECA}
	\includegraphics[width=0.9\textwidth]{LECA}
\end{figure}

\begin{figure}[H]
	\caption{Mitochondria are endosymbiotic bacteria}
	\label{fig:Mitochondria:are:endosymbiotic:bacteria}
	\includegraphics[width=0.9\textwidth]{MitochondriaEndosymbiotic}
\end{figure}

\begin{figure}[H]
	\caption{Host was an archaeon}
	\label{fig:host:archaeon}
	\includegraphics[width=0.9\textwidth]{HostArchaeon}
\end{figure}

\begin{figure}[H]
	\caption{Can distinguish genes of archaeal or bacterial ancestry}
	\label{fig:distinguish:genes:archaeal:bacterial}
	\includegraphics[width=0.9\textwidth]{thiergart2012}
\end{figure}

Mysteries remain
\begin{itemize}
	\item How did a bacterium get into an archaeon? There are two extremes in a range of theories.
	\begin{itemize}
		\item In the Outside-in Model, and archaean developed a membrane, Figure \ref{fig:outside:in1}, which became invaginated. It enveloped mitochondria in a mutualistic association--Figure \ref{fig:outside:in2}. Later it developed membranes to protect DNA from the activity (free oxygen) of the mitochondria--Figure \ref{fig:outside:in3}--until the nucleus was fully enclosed--Figure \ref{fig:outside:in4}.
		\item In the Inside-out Model--Figure \ref{fig:inside:out}--the cell becomes associated with arachaea, which are gradually walled in \cite{baum2014inside}.
	\end{itemize}
	\item How was a nucleus formed?
\end{itemize}

\begin{figure}[H]
	\caption{Outside-in Model}\label{fig:outside:in}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{ }\label{fig:outside:in1}
		\includegraphics[width=\textwidth]{OutsideIn1}
	\end{subfigure}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{ }\label{fig:outside:in2}
		\includegraphics[width=\textwidth]{OutsideIn2}
	\end{subfigure}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{ }\label{fig:outside:in3}
		\includegraphics[width=\textwidth]{OutsideIn3}
	\end{subfigure}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{ }\label{fig:outside:in4}
		\includegraphics[width=\textwidth]{OutsideIn4}
	\end{subfigure}
\end{figure}

\begin{figure}[H]
	\caption{Inside-Out Model}\label{fig:inside:out}
	\includegraphics[width=0.9\textwidth]{InsideOut}
\end{figure}

But still easier to solve than the origin of life!
\begin{itemize}
	\item New genomic data keep emerging
	\item There is a possibility of finding intermediates
\end{itemize}

\cite{javaux2003recognizing},  

\section{\Gls{gls:phylogenetics}}


\subsection{Using Phylogenetics to Travel in Time}
Lecturer: Bet{\"u}l Ka{\c c}ar

A phylogenetic tree is an hypothesis, not an established fact. Key Points:
\begin{itemize}
	\item A phylogenetic tree demonstrates evolutionary
	relationships among genes, proteins or organisms.
	\item In a phylogenetic tree, two species are closely
	related if they have a more recent shared ancestor.
\end{itemize}

\begin{itemize}
	\item $\approx 1.2$ million eukaryotes catalogued
	\item $\approx 8.9$ million eukaryotes present on Earth
	\item $>99\%$ of species have gone extinct thus severally limiting our starting pool
\end{itemize}

Tree Thinking and phylogeny
\begin{itemize}
	\item The evolutionary history of a group is its phylogeny
	\begin{itemize}
		\item Since we can’t travel back in time to identify
		common ancestors relationships of existing
		species must be estimated or inferred from data
	\end{itemize}
	\item The branching pattern in a phylogenetic tree illustrates how organisms evolved from a list of common ancestors.
	\item The evolutionary history of extant organisms can
	be understood in terms of their shared inheritance
	\begin{itemize}
		\item which extant species evolved from the same ancestor?
		\item how were ancestral traits modified in different lineages?
	\end{itemize}
\end{itemize}

\begin{figure}[H]
	\caption{Often there are many ways to model relationships, but generally the simplest will be most accurate.}\label{PhylogenicTree}
	\includegraphics[width=0.9\textwidth]{PhylogenicTree}
\end{figure}

\begin{figure}[H]
	\caption{Molecular Phylogenetics Protocol}\label{fig:MolecularPhylogeneticsProtocol}
	\includegraphics[width=0.9\textwidth]{MolecularPhylogeneticsProtocol}
\end{figure}
Some other applications of phylogenetics
\begin{itemize}
	\item Classification and taxonomy of genes, proteins and species
	\item Comparative analysis and character evolution
	\item Predicting protein structure and function
	\item Investigating the history and demography of populations
	\item Studying the emergence and spread of viral and bacterial pandemics
	\item Searching for useful traits in related groups
\end{itemize}

\begin{figure}[H]
	\caption{Phylogenetic Tree of Life--The big picture}
	\label{flg:the:big:picture}
	\includegraphics[width=0.9\textwidth]{TOL5}
\end{figure}

See \cite{hillis1996molecular}, \cite{zuckerkandl1965molecules}, \cite{williams2006assessing}, \cite{woese2002evolution}

\subsection{A Deeper Dive into Phylogenetics}

Lecturer: Andy Rominger
\begin{itemize}
	\item How to Infer a Phylogeny
	\item Why is the Deep Past so difficult to Infer?
\end{itemize}

\subsubsection{How to Infer a Phylogeny}
We are going to focus on step 3 of Figure \ref{fig:MolecularPhylogeneticsProtocol}, finding the evolutionary process leading to the data we have. We start with homologous sequences, and we want to infer the speciation, extinction, and passing genes on to daughter lineages.

We have transition rates, topology, and sequence data--Figures \ref{fig:InferringPhylogeny} \& \ref{fig:InferringPhylogeny1}, and we want to choose transition rates and a topology to \textit{maximize the likelihood}--Figures \ref{fig:InferringPhylogeny2}. 

\begin{figure}[H]
	\caption{Inferring a phylogeny}\label{fig:InferringPhylogeny}
	\includegraphics[width=0.9\textwidth]{InferringPhylogeny}
\end{figure}

\begin{figure}[H]
	\caption{Goal: find the process leading to the data}\label{fig:InferringPhylogeny1}
	\includegraphics[width=0.9\textwidth]{InferringPhylogeny1}
\end{figure}

\begin{figure}[H]
	\caption{Maximize Likelihood}\label{fig:InferringPhylogeny2}
	\includegraphics[width=0.9\textwidth]{InferringPhylogeny2}
\end{figure}

\begin{itemize}
	\item Maximum Likelihood Inference\cite{huelsenbeck1997phylogeny}
	\item Bayesian inference: traversing the complex topology and
	rate matrix space lends itself to Bayesian algorithms\cite{huelsenbeck2001mrbayes,huelsenbeck2001introduction}
\end{itemize}


Two other means have been used historically:
\begin{itemize}
	\item Parsimony
	\item Distance based inference
\end{itemize}

\subsubsection{Why is the Deep Past so difficult to infer?}
Caveats.
\begin{itemize}
	\item Inferring the past it hard. Unrooted phylogeny arises because mutation process is reversible in our models--Figure \ref{fig:unrooted:phylogeny}. This is why the Banfield Tree of Life\cite{hug2016new}--Figure \ref{fig:banfield:tol}--has no root.
	\item Rooting the Tree with an Outgoup
	\item Time Calibration
	\item \Gls{gls:long:branch:attraction}
	\item What genetic information goes back to LUCA?
\end{itemize}



\begin{figure}[H]
	\caption{Banfield Tree of Life}\label{fig:banfield:tol}
	\includegraphics[width=0.9\textwidth]{TOL4}
\end{figure}

\begin{figure}[H]
	\caption{Unrooted phylogeny arises because mutation process is reversible in our models}\label{fig:unrooted:phylogeny}
	\begin{subfigure}[b]{0.45\textwidth}
		\includegraphics[width=\textwidth]{InferringPastHard1}
	\end{subfigure}
	\begin{subfigure}[b]{0.45\textwidth}
		\includegraphics[width=\textwidth]{InferringPastHard2}
	\end{subfigure}
\end{figure}

The best method for finding a root is to use an \gls{gls:outgroup}--Figure \ref{fig:bird:outgroup}--which then gives a temporal direction to the tree: the tree flows away from the hypothetical common ancestor. But the Tree of [all] Life, by definition, has no outgroup!

\begin{figure}[H]
	\caption{Using a Bird as an Outgroup for the Tree of Mammals}\label{fig:bird:outgroup}
	\includegraphics[width=0.9\textwidth]{Outgroup}
\end{figure}

Another problem is that rate heterogeneity means that mutations are not clock like\cite{sanderson2003r8s,smit2007evolutionary}. Time in a phylogeny is measured in substitutions, not in years. Since the rate of mutation varies across lineages and across Earth's history, mutation events don't tell us the time on each branch. So species in the Banfield Tree of Life--Figure \ref{fig:banfield:tol}-- don't line up.
\begin{itemize}
	\item We may be able to assign times from the fossil record.
	\item In some cases we \textit{do} know the mutation rate and can use it like a clock.
\end{itemize}

Long branches attract each other\cite{philippe2005heterotachy}--Figure \ref{fig:long:branches}.

\begin{figure}[H]
	\caption{Long branches attract each other}\label{fig:long:branches}
	\begin{subfigure}[b]{0.45\textwidth}
		\includegraphics[width=\textwidth]{LongBranchAttraction1}
	\end{subfigure}
	\begin{subfigure}[b]{0.45\textwidth}
		\includegraphics[width=\textwidth]{LongBranchAttraction2}
	\end{subfigure}
\end{figure}

 What genetic information goes back to LUCA--Figure \ref{fig:back_to_LUCA}? Ribosomal RNA and protein genes\cite{quast2012silva} and Figure \ref{fig:Ribosome-5-3-2}.
 
 \begin{figure}[H]
 	\caption{What genetic information goes back to LUCA? }\label{fig:back_to_LUCA}
 	\includegraphics[width=0.9\textwidth]{LUCA_what_goes_back}
 \end{figure}

\begin{figure}[H]
	\caption{What genetic information goes back to LUCA? Ribosomal RNA and protein genes}\label{fig:Ribosome-5-3-2}
	\includegraphics[width=0.9\textwidth]{Ribosome-5-3-2}
\end{figure}


\section{Macroscopic Theories in Biology}

Lecturer: Chris Kempes

\begin{itemize}
	\item Which aspects of extant life are general, and which are arbitrary?
	\item How much of what we see is peculiar to the trajectory of life on our own planet?
	\item How much would be true of life anywhere?
\end{itemize}

This matters not just for exobiology, but also for understanding early life and the origin of life in general.

In this course we have covered:
\begin{itemize}
	\item Laws of chemistry
	\item General processes of natural selection
	\item Laws of physics
	\item Laws of Life?
\end{itemize}

As an example of Laws of Life, Figure \ref{fig:allometric:scaling} shows the Relationship between peak postfeeding resting metabolic rate and body mass for mammals\cite{white2005allometric}--an approximate power law with an exponent of $\frac{3}{4}$. West, Brown and Enquist\cite{west1997general} have shown how this law follows from the fractal vascular structure of mammals--Figure \ref{fig:FractalBranching}. Optimize so it fills space and bring nutrients to all cells, taking into account hydrodynamics, then wh get the $\frac{3}{4}$ exponent. The same thing works for plants. So Body Plan + Constraints$\rightarrow$Law of Life.

\begin{figure}[H]
	\caption{Relationship between peak postfeeding resting metabolic rate and body mass.}\label{fig:allometric:scaling}
	\includegraphics[width=0.9\textwidth]{WhiteSeymour}
\end{figure}

\begin{figure}[H]
	\caption{Power Law follows from the fractal vascular structure of mammals.}\label{fig:FractalBranching}
	\includegraphics[width=0.9\textwidth]{FractalBranching}
\end{figure}

Example of a Fundamental Constraint: diffusion to a simple spherical cell drifting passively. Cell is living in a still fluid filled with nutrients, and we want to know how nutrients diffuse to cell. If we assume that cell uses all of nutrient, so concentration at surface is held at zero.

\begin{align*}
C(r) =& C_{\infty}\big(1 - \frac{r_C}{r}\big)\text{, where}\\
C_{\infty} =& \text{ concentartion at infinity}\\
C(r) =& \text{ concentration at distance r}\\
J(r) =& D\frac{\partial C}{\partial R} \text{ flux}\\
  =& D C_{\infty} \frac{r_c}{r^2}\\
U =& 4 \pi r_C^2 J(r_C) \text{ total uptake of nutrient}\\
=&4 \pi D C_{\infty} r_C 
\end{align*}

No rate process within the cell can use the nutrient more quickly than $U$, so all processes must scale with $r$ at this rate or more slowly. If process want to go faster than $U$, the fluid needs to be disturbed, or the cell needs to move.

Figure \ref{fig:BacterialPhysiology} shows the variation of component size with cell size\cite{kempes2016evolutionary}; it \textit{bounds} the cell size: at the large end and small end the cell runs out of space for components.

\begin{figure}[H]
	\caption{BacterialPhysiology}\label{fig:BacterialPhysiology}
	\includegraphics[width=0.9\textwidth]{BacterialPhysiology}
\end{figure}

We could see how the fundamental physiological constraint change for other conceivable forms of life.

See also \cite{kempes2011predicting}.

\section{Selection}

Lecturer: Michael Lachmann

''A naturalist, reflecting on the mutual affinities of organic beings, on their embryological relations, their geographical distribution, geological succession, and other such facts, might come to the conclusion that each species had not been independently created, but had descended, like varieties, from other  species''--Charles Darwin\cite{darwin1859origin}.

Darwin\cite{darwin1859origin} presented a phylogenetic Tree--\ref{fig:TOL:Darwin}. A modern view, Figure \ref{fig:TOL:Modern} is more like a web.

\begin{figure}[H]
	\caption{Two views of the Tree of Life}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{After Darwin}\label{fig:TOL:Darwin}
		\includegraphics[width=\textwidth]{TOL_Darwin}
	\end{subfigure}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{Modern TOL}\label{fig:TOL:Modern}
		\includegraphics[width=\textwidth]{TOL-5-6}
	\end{subfigure}
\end{figure}

''Nevertheless, such a conclusion, even if well founded, would be unsatisfactory, until it could be shown how the innumerable species inhabiting this world have been modified, so as to acquire that perfection of structure and coadaptation which most justly excites our admiration''--\textit{ibid}.

How species acquire that perfection of structure:
\begin{itemize}
	\item Variation
	\item Inheritance
	\item Selection
\end{itemize}

\gls{gls:price:equation}--(\ref{eq:price:equation})

\begin{align*}
\Delta Z =& \frac{Cov(Z,W) + E(W \Delta Z)}{W}\text{, where} \numberthis \label{eq:price:equation}\\
Z=& \text{ some property, e.g. colour,}\\
W=& \text{ survival.}\\
 & \textit{Rearranging, }\\
W \Delta Z =& \underbrace{Cov(Z,W)}_\text{selection} + \underbrace{E(W \Delta Z)}_\text{transformation}
\end{align*}

How species acquire that perfection of structure? Because traits increase in value if they help individuals who have them survive, compared to those they don't.

\section{Selection Theory}
Chris Kempes

\subsection{Quasispecies equation}
\begin{itemize}
	\item How do we think about evolution in a simple, mathematical way?
	\item Start with binary Genome. One string can change into another via point mutations.
\end{itemize}


\begin{figure}[H]
	\caption{Topology of genome -- point mutations represent edges}\label{fig:GenomeTopology} 
	\includegraphics[width=0.9\textwidth]{GenomeTopology}
\end{figure}

\begin{figure}[H]
	\caption{Fitness Landscape, showing maximum fitness.}\label{fig:FitnessLandscape} 
	\includegraphics[width=0.9\textwidth]{FitnessLandscape}
\end{figure}

Introduce \emph{Quasispecies equation}.
\begin{align*}
\dot x_i =& f_i x_i - \phi x_i \text{, where $x_i$ is frequency of sequence $i$, $f_i$ is fitness, $\phi$ is death term.}\\
\sum_{i=1}^{n} \dot x_i =& 0 \text{, assume constant population}\\
\sum_{i=1}^{n} f_i - \phi \sum_{i=1}^{n} x_i =& 0\\
\phi =& \sum_{i=1}^{n}x_i f_i\text{, since $\sum_{i=1}^{n}x_i = 1$}
\end{align*}
Hence $\phi$ is \emph{average fitness}. But this leaves out \emph{mutation!} Include $q_{ij}$, the probability of mutation from $i$ to $j$.
\begin{align*}
\dot x_i =& \sum_{j=1}^{n} f_j q_{ij}x_j - \phi x_i\text{, full quasispecies equation} 
\end{align*}

NB. See Figure \ref{fig:GenomeTopology} - point mutations much more likely than 2-point, etc.

\subsection{What are the basic ways in which we think about evolution?}

\begin{itemize}
	\item How do we generalize evolution?
	\item How do we use this to think about origins of life? 
\end{itemize}

\begin{figure}[H]
	\caption[Simplify to \emph{Master Sequence}]{Simplify to \emph{Master Sequence}, $f_0$ with all less fit sequences lumped as $f_1$.}\label{fig:SteadyStateSolution} 
	\includegraphics[width=0.9\textwidth]{SteadyStateSolution}
\end{figure}

\begin{align*}
\dot x_i =& \sum_{j=1}^{n} f_j q_{ij}x_j - \phi x_i\\
f_0 >&1, f_1 = 1\\
q_{00} =& (1-u)^L \triangleq q\text{, where $u$ is mutation rate, $L$ = length of genome}\\
q_{01} =& 1-q\\
q_{11} =& 1\text{, $L$ is large, so 1$\rightarrow$1} \\
q_{10} =& 0\text{, $1\nrightarrow0$}\\
\phi =& \sum_{i=1}^{n}x_i f_i\\
\text{Whence,}\\
\dot{x_0}=&f_0 q x_0 - \phi x_0\text{,}\\
\dot{x_1}=&f_0 (1-q) x_0 + x_1 -\phi x_1\text{, and}\\
\phi =& f_0 x_0 + x_1 \text{. But} \\
x_0 + x_1 =& 1\text{, so we have}\\
\dot{x_0}=& x_0 \big[f_0 q -1 - x_0 (f_0 - 1)\big]
\end{align*}
Now we can look for a steady state solution:
\begin{align*}
x_0^* =& \frac{f_0 q - 1}{f_0 - 1}\text{, se we need}\\
f_0q >& 1 \text{, if the master sequence is to persist. I.e.}\\
u <& log(f_0)\\
log(f_0) >& -L log(1-u)\text{. Now we assume $L<<1$, and find}\\
u <& \frac{log(f_0)}{L}\text{, which is known as the \emph{Error Threshold.}} 
\end{align*}
The Error Threshold is the \emph{maximum mutation rate that allows for adaptive evolution!} NB: it is more sensitive to $L$ than to $f_0$.

\subsection{Quasispecies and Error Catastrophe}

Michael Lachmann

We look at growth of types, e.g.:
\begin{itemize}
	\item Genome in population
	\item Molecule in test-tube.
\end{itemize}

\begin{align*}
\vec{x}^{\,t} =& W M \vec{x}\text{, where M represents mutations, W growth, e.g.}\\
M =& \begin{bmatrix}
1-\mu & 0\\
\mu& 1
\end{bmatrix}&
\end{align*}
Let us diagonalize WM
\begin{align*}
WM =& VDV^{-1}\\
(WM)^t =& V D^t V^{-1}
\end{align*}
So $D^t$ is dominated by its largest eigenvalue.

Figures \ref{fig:ErrorCatastrophe1} and \ref{fig:ErrorCatastrophe2} show that master sequence lost if mutation rate is too high. 
\begin{figure}[H]
	\caption{Hamming Distance.}\label{fig:ErrorCatastrophe1} 
	\includegraphics[width=0.9\textwidth]{ErrorCatastrophe1}
\end{figure}
\begin{figure}[H]
	\caption{Error Catastrophe.}\label{fig:ErrorCatastrophe2} 
	\includegraphics[width=0.9\textwidth]{ErrorCatastrophe2}
\end{figure}


See \cite{eigen1978hypercycle}, \cite{eigen1988molecular}, \cite{eigen2002error}, \cite{crotty2001rna}, and \cite{stadtler2002fitness_landscapes}.

See also \cite{wessner2010origins}


\section{Artificial Life Theory}

Lecturer: Sara Imari Walker

Figure \ref{fig:PhysicsvsBiology} compares the evolution of a physical system with a biological system. Even the laws evolve in biology. Also biological systems involve homeostasis.
\begin{figure}[H]
	\caption{Physics: fixed law of evolution to final State, vs. Biology}\label{fig:PhysicsvsBiology}
	\includegraphics[width=0.9\textwidth]{PhysicsvsBiology}
\end{figure}

Von Neumann's work on self reproducing automata \cite{neumann1966theory}  \cite{neumann1958computer} is analogous to  the architecture of a modern cell--Figure \ref{fig:VonNeumann}. Note the instructional Tape in Figure  \ref{fig:VonNeumann1}--a small part of a larger biological algorithm. Note also that Von Neumann's work on cellular automata largely preceded our understanding of the translation from DNA to protein.



\begin{figure}[H]
	\caption{Von Neumann's work on self reproducing automata is analogous to the architecture of a modern cell}\label{fig:VonNeumann}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{The ribosome + assisting biomolecules
			act like a Universal Constructor}\label{fig:VonNeumann1}
		\includegraphics[width=\textwidth]{VonNeumann1}
	\end{subfigure}
	\begin{subfigure}[b]{0.45\textwidth}
		\caption{Supervisory Unit}\label{fig:VonNeumann2}
		\includegraphics[width=\textwidth]{VonNeumann2}
	\end{subfigure}
\end{figure}

“physical” universality: the ability to implement any transformation whatsoever on any finite region \cite{janzing2010there}, \cite{schaeffer2014physicallyuniversal}

Are the ’laws of life’  the laws of information?
% end of text 

% glossary
\printglossaries

% bibliography go here
 
\bibliographystyle{unsrt}
\addcontentsline{toc}{section}{Bibliography}
\bibliography{origins,wikipedia}

\end{document}
